rental.rb

```diff
@@ -0,0 +1,14 @@
+# class for rental
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] book and person should also be a getter and setter variable since they can be read and modified so make it accessible with attr_accessor